src/docs/manual/index.html

```diff
@@ -1118,91 +1128,191 @@ <h2 id="command-line-options">Command-line options</h2>
+The code under test can make calls to methods that you do not want executed.
+One is <code>System.exit</code>, which will terminate Randoop if called during test generation.
+Others, such as the <code>JOptionPane.showConfirmDialog</code> methods, open a modal dialog box requiring user interaction, which interferes with automated testing.
+It is not enough to use <a href="#option:omitmethods"><code>--omitmethods</code></a> to
+prevent direct calls to these methods, because the methods may still be called indirectly.
+To prevent all calls to these methods, use the <code>mapcall</code> agent.
 </p>
 
 <p>
-You can prevent Randoop from calling certain methods in the code under
-test, by using the
-<code>--omitmethods=regexp</code> command-line option.
+The Randoop <code>mapcall</code> agent transforms the code under test, replacing
+each call to an undesired method by a call to some other method.
+To use it:
 </p>
+<ul>
+  <li>When running Randoop, pass the following command-line options to <code>java</code>:
+<pre>
+    -Xbootclasspath/a:$(RANDOOP_PATH)/mapcall-3.1.5.jar -javaagent:$(RANDOOP_PATH)/mapcall-3.1.5.jar
+</pre>
+  </li>
+  <li>When running the generated tests, add the file <code>mapcall-3.1.5.jar</code>,
+    and pass the same command-line options to <code>java</code>:
+<pre>
+    -Xbootclasspath/a:$(RANDOOP_PATH)/mapcall-3.1.5.jar -javaagent:$(RANDOOP_PATH)/mapcall-3.1.5.jar
+</pre>
+</li>
+</ul>
+
+<p>
+By default, the agent makes the following replacements:
+</p>
+<ul>
+  <li>
+    A call to <code>System.exit()</code> is replaced by a call to a method that
+    throws a Randoop-specific exception that Randoop includes in the generated tests.
+  </li>
+  <li>
+    A call to certain methods of AWT and Swing classes is replaced by a call to
+    an implementation that is designed to avoid hanging both during test generation
+    by Randoop and during execution of the generated tests.
+  </li>
+</ul>
+<p>
+In addition, Randoop treats replaced methods as if the methods were given as arguments
+to <a href="#option:omitmethods"><code>--omitmethods</code></a> and does not create
+direct calls to them in test cases.
+</p>
+
+
+
+<h3 id="mapcall-load-exclusions">Replacement exclusions</h3>
+
+<p>
+There are some classes to which the mapcall agent should not be applied.
+</p>
+
+<p>
+If you are using the agent in a build environment, it is also necessary to exclude the build environment classes.
+For instance, when running the agent on generated tests in a Gradle project using the Java plugin, the replacement of <code>System.exit</code>
+will affect the behavior of the build, and we should exclude the Gradle packages from being transformed.
+To exclude these packages, an exclusion file <code>gradle-exclusions.txt</code> is created with the lines
+</p>
+<pre>
+org.gradle.
+worker.org.gradle.
+</pre>
+<p>
+Then the agent is run with the <code>--load-exclusions</code> command-line option with the filename as the argument:
```

Comment:
Not only is is odd, it is wrong.  The actual argument name is `--dont-transform`.